src/include/hip/rpp_hip_common.hpp

```diff
@@ -82,6 +82,7 @@ typedef union { uchar uc1[8];   uchar4 uc4[2];
 typedef union { uchar uc1[24];  uchar4 uc4[6];  uchar3 uc3[8];    d_uchar8 uc8[3];              }   d_uchar24;
 
 // schar
+typedef struct { schar sc1[3];                                                                  }   d_schar3_s;
```

Comment:
There is schar3 available in hip api i think
Please check once

Reply:
HIP doesn't support schar3 as inbuilt datatype, thus introduced this struct.